src/bundle/Resources/translations/content.en.xliff

```diff
@@ -11,6 +11,11 @@
         <target state="new">New Version Draft for '%name%' created.</target>
         <note>key: content.create_draft.success</note>
       </trans-unit>
+      <trans-unit id="d773dd487e4e5467fd20fc4fc4987eba33eaf1b7" resname="content.main_language_update.success">
+        <source>Main language for '%name%' updated.</source>
+        <target state="new">Main language for '%name%' updated.</target>
```

Comment:
Please, remove `state="new"` from here.

Reply:
I think we had this disscusion and that info is useful for some people.

Reply:
we talked about it and AFAIK we decided to not remove this information